packages/api/src/test-server.ts

```diff
@@ -58,6 +58,7 @@ if (!params.insecureTestToken) {
 }
 params.listen = true;
 params.requireEmailVerification = true;
+params.isTest = true;
```

Comment:
This is redundant, we should just use the `NODE_ENV` environment variable when we need to know if it is a test. e.g. https://github.com/livepeer/livepeer-com/blob/1fd399ff3387dd915a8b7a3ac36e123ce17af64b/packages/api/src/app-router.ts#L125-L126